src/bosh-director/lib/bosh/director/jobs/update_release.rb

```diff
@@ -304,18 +305,18 @@ def use_existing_jobs(jobs, release_dir)
         return false if jobs.empty?
 
         single_step_stage("Processing #{jobs.size} existing job#{'s' if jobs.size > 1}") do
-          jobs.each do |template, job_meta|
-            job_desc = "#{template.name}/#{template.version}"
+          jobs.each do |job_model, manifest_job|
+            job_desc = "#{job_model.name}/#{job_model.version}"
```

Comment:
At some point maybe the "#{name}/#{version}" moniker should be extracted into a method rather than having this pattern of string interpolation.

I don't think this is necessary for this PR but I thought it was worth including a comment in case there are more refactors coming.

Reply:
Yep, I agree with this indeed